src/matchers/toBeBetween/predicate.test.js

```diff
@@ -1,15 +0,0 @@
-import predicate from './predicate';
-
-const TESTDATE1 = new Date('01/09/2019');
```

Comment:
_index.test.js_ has all the dates formatted in ISO 8601. So deleting this file deletes the only tests with this format. Not sure if we care about that.